README.md

```diff
@@ -42,3 +42,19 @@ mvn clean install -DfailIfNoTests=false -Pintegration -Dit.test=NrtmClientInvali
 
 mvn clean install -Pintegration -pl whois-query
 
+
+## Configure Ajc (AspectJ) Compiler
+
+Whois uses AspectJ to perform compile-time aspect weaving of the code. This is needed for some functionality, e.g. the @RetryFor annotation.
+
+Compile-time weaving works during a command-line Maven build, as the pom.xml uses aspectj-maven-plugin.
+
+Any code that depends on AspectJ will fail if modified in IntelliJ without using the Ajc (AspectJ) compiler.
+
+You can configure Ajc in the Preferences as follows:
+
```

Comment:
```suggestion

* First install the AspectJ plugin provided by JetBrain from the IDE plugin repository
```
This is required, otherwise the Ajc compiler won't be available as an option.